tests/e2e/package.json

```diff
@@ -7,56 +7,25 @@
     "cleanup-docker": "if [ $(docker ps -a | grep -c selenium-e2e) -gt 0 ]; then docker rm -f $(docker ps --filter \"name=selenium-e2e\" -aq); fi;",
-    "init-mocha-opts": "tsc && mocha --config mocha-single-devfile.json --spec dist/tests/login/Login.spec.js",
     "test-docker": "npm run cleanup-docker && docker run -it --shm-size=2g -p 5920:5920 --name selenium-e2e -e TS_SELENIUM_BASE_URL=$TS_SELENIUM_BASE_URL eclipse/che-e2e:nightly",
     "test-docker-mount-e2e": "npm run cleanup-docker && docker run -it --shm-size=2g -p 5920:5920 --name selenium-e2e -e TS_SELENIUM_BASE_URL=$TS_SELENIUM_BASE_URL -v $(pwd):/tmp/e2e:Z eclipse/che-e2e:nightly",
-    "test-oauth": "tsc && mocha --config mocha.ocp.link.json",
-    "test-intelij": "tsc && mocha --config mocha.intelij.json",
-    "test-plugin": "./initPluginTest.sh",
-    "test-all-devfiles": "./generateIndex.sh && ./initDevfileTests.sh",
-    "test-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-theia.json",
-    "load-test-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-load-theia.json",
-    "test-happy-path-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-happy-path-theia.json",
-    "test-devworkspace-happy-path-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-devworkspace-happy-path-theia.json",
-    "test-ws-creation-and-ls-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-ws-creation-and-ls-theia.json",
-    "test-java-vertx-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-java-vertx-theia.json",
-    "test-java-springboot-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-java-springboot-theia.json",
-    "test-git-ssh-theia": "./generateIndex.sh &&  npm run lint && npm run tsc && mocha --config mocha-git-ssh-theia.json",
-    "test-git-self-sign-cert-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-git-self-sign-cert-theia.json",
-    "test-git-publish-branch-theia": "./generateIndex.sh &&  npm run lint && npm run tsc && mocha --config mocha-git-publish-branch-theia.json",
-    "test-openshift-connector-theia": "./generateIndex.sh &&  npm run lint && npm run tsc && mocha --config mocha-connector-theia.json",
-    "test-factory-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-factory-theia.json",
-    "test-all-factories-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-all-factories-theia.json",
-    "test-plugin-ci-theia": "export TS_DELETE_PLUGINS_TEST_WORKSPACE=true && npm run init-mocha-opts -- --spec dist/tests/plugins/theia/${USERSTORY}.spec.js",
-    "test-all-plugins-theia": "tsc && mocha --config mocha-all-plugins-theia.json",
-    "test-code": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-che-code.json",
```

Comment:
Quite a bit of these `-code` configurations were removed and replaced with only three cases, is that intentional? Will it influence any of the scenarios that we launch?

Reply:
Now in Che we have only one userstory for Che-Code (and in general too) - EmptyWorkspace.spec.ts
All those configs from scripts had an empty "specs" and I did not find where we use them. Maybe we will add some more when we will have more suits like "smoke" and "regression"